### PR TITLE
AT_15.06_002 | Folder > Same name Error when renaming Folder with existing name by dropdown menu

### PR DESCRIPTION
**Pull request by verafes**
AT - https://trello.com/c/r64IblQK

Err - invalid extension -  https://trello.com/c/UdxEThHe

ERR - skip failed test - https://trello.com/c/LekksEVn

### CODE REVIEW

File: cypress/e2e/folderRenameFolder.cy.js [hunk 1]
```diff
@@ -27,4 +27,22 @@ describe('Folder rename folder', () => {
         cy.url().should('eq', jenkinsURL + '/job/' + createFolder.folderName + 'CHANGED/');
         cy.get('#main-panel > h1').should('contain', createFolder.folderName + 'CHANGED');
     });
+
+    it.only('AT_15.06_002 | Folder> Same name Error message if renaming Folder by dropdown menu' , () => {
```
Comment (Rom-T) [inline cypress/e2e/folderRenameFolder.cy.js:31]: Please delete `only`

File: cypress/e2e/folderRenameFolder.cy.js [hunk 2]
```diff
@@ -27,4 +27,22 @@ describe('Folder rename folder', () => {
         cy.url().should('eq', jenkinsURL + '/job/' + createFolder.folderName + 'CHANGED/');
         cy.get('#main-panel > h1').should('contain', createFolder.folderName + 'CHANGED');
     });
+
+    it.only('AT_15.06_002 | Folder> Same name Error message if renaming Folder by dropdown menu' , () => {
+        cy.get('#breadcrumbs > li:nth-child(1) > a').click();
+        cy.get('tbody tr td a.jenkins-table__link')
+            .should('be.visible')
+            .should('have.text', createFolder.folderName)
```
Comment (Rom-T) [inline cypress/e2e/folderRenameFolder.cy.js:35]: When we make several assertions for 1 element, only the first assertion should, and all subsequent ones are written as `and`

Comment (verafes) [reply cypress/e2e/folderRenameFolder.cy.js:35]: as what?

Comment (DariaMas7) [reply cypress/e2e/folderRenameFolder.cy.js:35]: cy.get('tbody tr td a.jenkins-table__link')
            .should('be.visible')
            .and('have.text', createFolder.folderName)

**Review by AnnaIurchyk — APPROVED** [review]
LGTM

**Review by Rom-T — APPROVED** [review]
Well done!

**Pull request merged**